webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -127,6 +128,7 @@ const HostDetails = ({
             hostName={id}
             status={status}
             tabs={tabs}
+            router={history}
```

Comment:
Why the name change here?

Reply:
I guess this change is made to distinguish between the main router and hash-router history.

Reply:
Then should it be called `hashRouter` instead?

Reply:
This history object represents the main router's history, not the `hashRouter` one